### DISCUSSION TITLE
Stargate SG1 The Alliance

### DISCUSSION THREAD

**Discussion by Emuroland** [EliotVU/Unreal-Library#88] [General]
Is there any chance somebody make this unfinished game compatible with Unreal-library/UE Explorer? Thank you!

**Comment by EliotVU** [comment]
Stargate Worlds? Should be compatible, perhaps it has regressed, I'll have a look.

**Reply by Emuroland** [reply]
Sorry, it is not stargate worlds, it is Stargate SG1 The Alliance cancelled fps.Ue2-ish engine, only Xbox classic version surfaced.
Thank you very much!

**Reply by EliotVU** [reply]
Oh nice, I grabbed a copy on the archive, and figured it's mostly compatible, there just happens to be a `StructFlags` added to the `UStruct` class that is breaking the alignment, easy fix.

**Comment by EliotVU** [comment]
Supported with https://github.com/EliotVU/Unreal-Library/commit/660c0c289db89f675a253bf799bb80baf4ec3c83
